src/core/server/saved_objects/service/saved_objects_client.test.js

```diff
@@ -54,6 +54,45 @@ test(`#bulkCreate`, async () => {
   expect(result).toBe(returnValue);
 });
 
+describe(`#createPointInTimeFinder`, () => {
+  test(`calls repository with options and default dependencies`, () => {
+    const returnValue = Symbol();
+    const mockRepository = {
+      createPointInTimeFinder: jest.fn().mockReturnValue(returnValue),
+    };
+    const client = new SavedObjectsClient(mockRepository);
+
+    const options = Symbol();
+    const result = client.createPointInTimeFinder(options);
+
+    expect(mockRepository.createPointInTimeFinder).toHaveBeenCalledWith(options, {
+      find: expect.any(Function),
+      openPointInTimeForType: expect.any(Function),
+      closePointInTime: expect.any(Function),
```

Comment:
I wanted to assert that the actual functions from the `client` were used here (e.g. `client.find`), but since they are bound to `this` inside the client, the equality check fails.

Tried to think of a way to mock around this but came up empty - open to suggestions!